libqtile/widget/memory.py

```diff
@@ -39,27 +39,33 @@ class Memory(base.ThreadedPollText):
     SwapFree: Returns amount of swap free
     SwapUsed: Returns amount of swap in use
 """
+
     orientations = base.ORIENTATION_HORIZONTAL
     defaults = [
-        ("fmt", "{MemUsed}M/{MemTotal}M", "Formatting for field names.")
+        ("format", "{MemUsed}M/{MemTotal}M", "Formatting for field names."),
```

Comment:
I don't think you should rename it `format`. It can shadow the built-in format function, making its use confusing. All other widgets also use `fmt`, this would break consistency. Wouldn't it be a breaking change as well, since users would need to update their code?

Reply:
Hmm, this is interesting. If the old method was broken, doing an api breaking change should be fine to fix it.

Reply:
It might be broken since the implementation of `fmt` in the base class. I'd be in favor of moving it to `format` so that it's consistent with other widgets, `fmt` being the "overall" format of the output and `format` being the internal formatting of the widget.